### PR TITLE
fix(ci): update submodule script

### PR DESCRIPTION
**Pull request by raisedadead**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: tools/ci-scripts/trigger-build.sh [hunk 1]
```diff
@@ -6,7 +6,8 @@ then
   git clone https://github.com/"$DOCKER_PUSH_REPO".git docker-push-repo
 
   cd docker-push-repo/freecodecamp
```
Comment (raisedadead) [inline tools/ci-scripts/trigger-build.sh]: Wait this may not be needed actually.

File: .travis.yml [hunk 2]
```diff
@@ -28,5 +28,5 @@ jobs:
 
     - stage: Artifacts builds
       if: branch = master && type != pull_request
-      script: bash trigger-build.sh
- 
+      script: bash tools/ci-scripts/trigger-build.sh
```
Comment (Bouncey) [inline .travis.yml]:
```suggestion
      script: bash tools/ci-scripts/trigger-build.sh
      install: skip
      before_script: skip
```

File: tools/ci-scripts/trigger-build.sh (hunk 1 above)
Comment (Bouncey) [inline tools/ci-scripts/trigger-build.sh]:
```suggestion
   cd docker-push-repo
```

File: tools/ci-scripts/trigger-build.sh [hunk 3]
```diff
@@ -6,7 +6,8 @@ then
...
-  git pull --rebase origin master
+  git submodule update --init --remote --recursive
```
Comment (Bouncey) [inline tools/ci-scripts/trigger-build.sh]: This works locally from the root of the repository

Comment (raisedadead) [reply tools/ci-scripts/trigger-build.sh]: Yes, I just realised that.

File: tools/ci-scripts/trigger-build.sh [hunk 4]
```diff
@@ -5,8 +5,9 @@ if [ -n "$DOCKER_PUSH_API_TOKEN" ]
 then
   git clone https://github.com/"$DOCKER_PUSH_REPO".git docker-push-repo
```
Comment (raisedadead) [inline tools/ci-scripts/trigger-build.sh]:
```suggestion

```

File: tools/ci-scripts/trigger-build.sh [hunk 5]
```diff
@@ -5,8 +5,9 @@ if [ -n "$DOCKER_PUSH_API_TOKEN" ]
...
 
-  cd docker-push-repo/freecodecamp
-  git pull --rebase origin master
+   cd docker-push-repo
```
Comment (raisedadead) [inline tools/ci-scripts/trigger-build.sh]:
```suggestion
  cd docker-push-repo
```

**Pull request merged**